eng/SourceBuild.props

```diff
@@ -37,6 +37,8 @@
       <InnerBuildArgs>$(InnerBuildArgs) /p:KeepNativeSymbols=true</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) /p:RuntimeOS=$(TargetRidWithoutPlatform)</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) /p:PortableBuild=$(SourceBuildPortable)</InnerBuildArgs>
+      <InnerBuildArgs Condition="'$(OfficialBuildId)' != ''">$(InnerBuildArgs) /p:OfficialBuildId=$(OfficialBuildId)</InnerBuildArgs>
+      <InnerBuildArgs>$(InnerBuildArgs) /p:ContinuousIntegrationBuild=$(ContinuousIntegrationBuild)</InnerBuildArgs>
```

Comment:
Curious, shouldn't this be handled by Arcade powered source build? Otherwise every consuming repository would need to add these.

Reply:
Runtime is special and overrides the base build command here: https://github.com/dotnet/runtime/pull/75293/files/d63395f9ee606cf5f6061122e29dfd3a96c31a1d#diff-6a28d92de8923be8821f2123541dc23c8ad0359c96117709e6d49e752c0303d2R9 which removes all the basic parameters.